document/endnotes_test.go

```diff
@@ -0,0 +1,285 @@
+package document_test
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/unidoc/unioffice/document"
+)
+
+func TestEndnotesLoad(t *testing.T) {
+	doc := loadTestData(t)
+	if doc == nil {
+		t.Fatal("could not read test document")
+	}
+	saveTestData(t, doc)
+}
+
+func TestHasEndnotes(t *testing.T) {
+	doc := loadTestData(t)
+	if doc == nil {
+		t.Fatal("could not read test document")
+	}
+	if doc.HasEndnotes() == false {
```

Comment:
This will work but does this way have any advantages over `if !doc.HasEndnotes()` ?